File: DTOs/Sources/RideServiceDTOs/CreateEventRideDTO.swift
import Foundation

public struct CreateEventRideDTO: Codable {
    public var eventID: UUID
    public var description: String?
    public var vehicleDescription: String?
    public var starts: Date
    public var latitude: Float
    public var longitude: Float
    public var emptySeats: UInt8
    
    public init(eventID: UUID, description: String? = nil, vehicleDescription: String? = nil, starts: Date, latitude: Float, longitude: Float, emptySeats: UInt8) {
        self.eventID = eventID
        self.description = description
        self.vehicleDescription = vehicleDescription
        self.starts = starts
        self.latitude = latitude
        self.longitude = longitude
        self.emptySeats = emptySeats
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/RideServiceDTOs/CreateEventRideDTO.kt
package net.ipv64.kivop.dtos.RideServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class CreateEventRideDTO (
    var eventID : UUID,
    var description : String?,
    var vehicleDescription : String?,
    var starts : LocalDateTime,
    var latitude : Float,
    var longitude : Float,
    var emptySeats : UByte,
)
